stdlib/planner/group_max_test.flux

```diff
@@ -0,0 +1,113 @@
+package planner_test
+
+import "testing"
+import "testing/expect"
+import "planner"
+
+option now = () => (2030-01-01T00:00:00Z)
+
+input = "
+#datatype,string,long,dateTime:RFC3339,string,string,string,double
+#group,false,false,false,true,true,true,false
+#default,_result,,,,,,
+,result,table,_time,_measurement,host,_field,_value
+,,0,2018-05-22T19:53:26Z,system,host.local,load1,1.83
+,,0,2018-05-22T19:53:36Z,system,host.local,load1,1.63
+,,1,2018-05-22T19:53:26Z,system,host.local,load3,1.72
+,,2,2018-05-22T19:53:26Z,system,host.local,load4,1.77
+,,2,2018-05-22T19:53:36Z,system,host.local,load4,1.78
+,,2,2018-05-22T19:53:46Z,system,host.local,load4,1.77
+"
+
+out_max_bare = "
+#datatype,string,long,dateTime:RFC3339,dateTime:RFC3339,dateTime:RFC3339,string,string,string,double
+#group,false,false,true,true,false,true,true,true,false
+#default,_result,,,,,,,,
+,result,table,_start,_stop,_time,_measurement,host,_field,_value
+,,0,2018-05-22T19:53:26Z,2030-01-01T00:00:00Z,2018-05-22T19:53:46Z,system,host.local,load4,1.77
+"
+
+out_max_window = "
+#datatype,string,long,dateTime:RFC3339,dateTime:RFC3339,dateTime:RFC3339,string,string,string,double
+#group,false,false,true,true,false,true,true,true,false
+#default,_result,,,,,,,,
+,result,table,_start,_stop,_time,_measurement,host,_field,_value
+,,0,2018-05-22T19:53:26Z,2030-01-01T00:00:00Z,2018-05-22T19:53:26Z,system,host.local,load4,1.77
+"
+
+out_max_agg_window = "
+#datatype,string,long,dateTime:RFC3339,dateTime:RFC3339,dateTime:RFC3339,string,string,string,double
+#group,false,false,true,true,false,true,true,true,false
+#default,_result,,,,,,,,
+,result,table,_start,_stop,_time,_measurement,host,_field,_value
+,,0,2018-05-22T19:53:26Z,2030-01-01T00:00:00Z,2018-05-23T00:00:00Z,system,host.local,load4,1.77
+"
+
+out_max_agg_window_empty = "
+#datatype,string,long,dateTime:RFC3339,dateTime:RFC3339,dateTime:RFC3339,string,string,string,double
+#group,false,false,true,true,false,true,true,true,false
+#default,_result,,,,,,,,
+,result,table,_start,_stop,_time,_measurement,host,_field,_value
+,,0,2018-05-22T19:53:26Z,2030-01-01T00:00:00Z,2018-05-23T00:00:00Z,system,host.local,load1,1.83
+,,1,2018-05-22T19:53:26Z,2030-01-01T00:00:00Z,2018-05-23T00:00:00Z,system,host.local,load3,1.72
+,,2,2018-05-22T19:53:26Z,2030-01-01T00:00:00Z,2018-05-23T00:00:00Z,system,host.local,load4,1.78
+"
+
+group_max_bare = () =>
+    testing.loadStorage(csv: input)
+    |> range(start: 2018-05-22T19:53:26Z)
+    |> filter(fn: (r) => r["_value"] == 1.77)
+    |> group(columns: ["_field"])
+    |> max()
+
+group_max_window = () =>
+    testing.loadStorage(csv: input)
```

Comment:
Hmm I see that `testing.loadStorage` requires that you pass in `csv` text.  What if we added a new function to the testing package like this?

```
option load = (tables=<-) => tables
    |> range(start: 1800-01-01T00:00:00Z, stop: 2200-12-31T11:59:59Z)
    |> map(fn: (r) => ({r with
    _field: if exists r._field then r._field else die(msg: "test input table does not have _field column"),
    _measurement: if exists r._measurement then r._measurement else die(msg: "test input table does not have _measurement column"),
    _time: if exists r._time then r._time else die(msg: "test input table does not have _time column")
    }))
  ```
  
  This would require that we make sure the drivers update the new `load` option in `influxdb` and `idpe`. Thoughts?

Reply:
Yeah this doesn't seem to difficult to achieve. Do we want this issue as a part of the acceptance test PR?